ui/src/assets/images/404.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="-204 496.5 416.5 198" style="enable-background:new -204 496.5 416.5 198;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#00456E;}

```

Comment:
you wot?

Reply:
Here it is minified, but it won’t keep size if not enclosed within a CSS element (me too dumb to know if it is already or not):
```
<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" viewBox="0 0 416.5 198"><defs><style>.cls-1{fill:#00456e;}</style></defs><path id="a" class="cls-1" d="M70.9,195.2h56.4v-2.8l-26.4-.3V135.4h34.5v-3.1H100.9V2.6H97.8L0,132.4v3.1H97.3v56.7l-26.4.3v2.7Zm-67-62.8v-.3L97,8.7h.3V132.4Z"/><path id="b" class="cls-1" d="M208.4,197.7c46.1,0,71.2-37,71.2-99S254.6,0,208.4,0s-71.2,37-71.2,98.7S162.2,197.7,208.4,197.7Zm0-3.1c-44.2,0-67.3-35.9-67.3-95.9S164.2,3.1,208.4,3.1s67.3,35.3,67.3,95.6-23.1,95.9-67.3,95.9Z"/><path id="c" class="cls-1" d="M352,195.2h56.4v-2.8l-26.4-.3V135.4h34.5v-3.1H382V2.6h-3.1L281.1,132.4v3.1h97.3v56.7l-26.4.3Zm-67-62.8v-.3L378.1,8.7h.3V132.4Z"/></svg>
```

Reply:
why is there a style tag in there tho? should be inlined or css really (will trigger a repaint)

Reply:
Sorry, too many options for the layman. Like that?
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 416.5 198"><path d="M70.9 195.2h56.4v-2.8l-26.4-.3v-56.7h34.5v-3.1h-34.5V2.6h-3.1L0 132.4v3.1h97.3v56.7l-26.4.3zm-67-62.8v-.3L97 8.7h.3v123.7zM208.4 197.7c46.1 0 71.2-37 71.2-99S254.6 0 208.4 0s-71.2 37-71.2 98.7 25 99 71.2 99zm0-3.1c-44.2 0-67.3-35.9-67.3-95.9s23.1-95.6 67.3-95.6 67.3 35.3 67.3 95.6-23.1 95.9-67.3 95.9zM352 195.2h56.4v-2.8l-26.4-.3v-56.7h34.5v-3.1H382V2.6h-3.1l-97.8 129.8v3.1h97.3v56.7l-26.4.3zm-67-62.8v-.3L378.1 8.7h.3v123.7z" fill="#00456e"/></svg>```

Using new Illustrator Export for Screens (+SVGOMG):
![picture 100](https://user-images.githubusercontent.com/6032869/30977570-d088460e-a46f-11e7-8832-5d3e8a6dc182.png)